src/sdk/endpoint_stats_collector.js

```diff
@@ -350,6 +353,16 @@ class EndpointStatsCollector {
             this.prom_metrics_report.observe('hub_write_latency', { bucket_name }, hub_write_latency);
         }
     }
+    update_fork_counter() {
+        // add fork related metrics to prometheus
+        const code = `worker_${cluster.worker.id}`;
+        this.prom_metrics_report.inc('fork_counter', {code});
+    }
+}
+if (cluster.isWorker) {
```

Comment:
can you explain about this change? not sure why we increase this counter every time minute

Reply:
@romayalon I added these metrics to get the number of forks running at a time. I am ok with removing these metrics.

Reply:
it's ok to have this metric, but wouldn't it increase the fork count every second? not sure how it's working

Reply:
yes, it will increase the count each time the fork runs, in this case, every minute, and the count doesn't have any significance. I am not sure Prometheus metrics can have a key without any value.

Reply:
removed setIntervel() now the code runs only once.